behaviour/concept/type/relationtype.feature

```diff
@@ -454,7 +460,7 @@ Feature: Concept Relation Type and Role Type
       | license     |
       | certificate |
 
-  Scenario: Relation types cannot have keys of attributes that are not keyable
+  Scenario: Relation types can have keys of all keyable attributes
```

Comment:
This change splits this test into two tests; one for the keyable attributes, and one for the non-keyable ones.